modules/party/Classes/Party.php

```diff
@@ -390,37 +399,51 @@ public function delete_usergroups($del_group, $set_group)
         $database->query("UPDATE %prefix%user SET group_id = ? WHERE group_id = ?", [$set_group, $del_group]);
         $database->query("DELETE FROM %prefix%party_usergroups WHERE group_id = ?", [$del_group]);
     }
-    
+
     /**
      * Returns the amount of users registered for a party.
-     * 
-     * @param int $party_id The ID of the party to calculate this for
+     *
+     * @param int $partyId The ID of the party to calculate this for (uses object value otherwise)
+     * @param
      * @return array Result array with elements "qty" and "paid"
     */
-    public function getGuestQty($party_id = NULL)
+    public function getGuestQty($partyId = null, $showOrga = null)
```

Comment:
Did we ever use the second parameter `$showOrga`?

Reply:
This _will_ be used, but is not at this point in time.
There is an config option to include registrations from admins/superadmins in the overview and I've added this to ensure that the function covers both cases. 
This is inteded to replace multiple calculations of similar type (if I remember correctly). 
If it turns out after that that it is not needed, then that's surely fine to remove